expense/package.json

```diff
@@ -1,14 +1,24 @@
 {
+  "name": "temporal-expense",
   "version": "0.1.0",
   "private": true,
-  "name": "temporal-expense",
   "scripts": {
-    "build": "tsc --build ./tsconfig.json",
-    "build.watch": "tsc --build --watch ./tsconfig.json",
-    "server": "node ./lib/server",
-    "starter-approve": "node ./lib/starter/approve",
-    "starter-timeout": "node ./lib/starter/timeout",
-    "worker": "node ./lib/worker"
+    "build": "tsc --build",
+    "build.watch": "tsc --build --watch",
+    "server": "ts-node src/server",
+    "start": "ts-node src/worker.ts",
+    "start.watch": "nodemon src/worker.ts",
+    "workflow-approve": "ts-node src/execute/approve",
+    "workflow-timeout": "ts-node src/execute/timeout"
+  },
+  "nodemonConfig": {
```

Comment:
Maybe we do a shared [`nodemon.json` file](https://github.com/remy/nodemon#config-files) as well?